recipes/libpq/all/test_package/CMakeLists.txt

```diff
@@ -1,6 +1,6 @@
 cmake_minimum_required(VERSION 3.1)
 project(test_package LANGUAGES C)
-find_package(PostgreSQL REQUIRED)
+find_package(PostgreSQL REQUIRED CONFIG)
```

Comment:
```suggestion
find_package(PostgreSQL REQUIRED)
```

Reply:
This doesn't work

Reply:
@SpaceIm If we want to avoid having the `CONFIG` option, I can move away from using the `add_subdirectory` command in the v1 CMakeLists file

Reply:
There is no reason it cannot work. If it doesnt work, something os broken in recipe.

Reply:
That's not true. If you look at CMake's docs on how `find_package` resolution works, this is expected behavior. Both the `cmake_find_package` and `cmake_find_package_multi` generators create config files. CMake has a `FindPostgreSQL.cmake` module. The module will take precedence over the generated config. I'll push a change to remove the `CONFIG` option, but the recipe will have to be different from the templates.

Reply:
> Both the cmake_find_package and cmake_find_package_multi generators create config files.

no

Reply:
Ok. I'll have to check why it isn't generating the module atm. I only get PostgreSQLConfig.cmake files. Any ideas?

Reply:
You have to replace `cmake_find_package_multi` by `cmake_find_package` in conanfile of test_v1_package as I said.